docs/docs/scheme-intro/next-step.md

```diff
@@ -38,7 +38,7 @@ There are also more recent lectures from 2010 at [Berkeley](https://www.berkeley
 
 ## Lisp Macros
 
-If you want to learn more about lisp macros, there are two great books:
+For more on Lisp macros, these two books are highly recommended:
```

Comment:
Same, I don't like the "are highly recommended" and changing the sentence.

This doesn't look like a grammar fix.